api/v1alpha1/freight_types.go

```diff
@@ -29,6 +29,12 @@ type Freight struct {
 	// present, the defaulting webhook will choose an available alias and assign
 	// it to both the field and label.
 	Alias string `json:"alias,omitempty" protobuf:"bytes,7,opt,name=alias"`
+	// Warehouse is the name of the Warehouse that created this Freight. This is a
+	// required field. TODO: It is not clear yet how this field should be set in
+	// the case of user-defined Freight.
+	//
+	//+kubebuilder:validation:Required
```

Comment:
Nit:

```suggestion
	// +kubebuilder:validation:Required
```

Reply:
Taking the suggestion, but the missing space between `//` and `+` is pervasive in this package.

I know the reason for this is that the first few types were created using `kubebuilder create api ...` and it didn't include that space. I blindly continued followed that precedent.

I'm unopposed to a follow-up chore to change this more broadly. idk why, but I strongly agree it's nicer to look at with the space included.

Reply:
Ah, it has been ages since I set up things using `kubebuilder`. In the past, once we had bootstrapped a project I never looked back and just wrote new API structures by hand. But this explains a lot.

Reply:
> In the past, once we had bootstrapped a project I never looked back and just wrote new API structures by hand.

That's the same route we've taken. Haven't touched it in forever... until last night when I generated a test CRD to verify that was where this weird precedent initially came from.